sphinx/tutorial/cylc/furthertopics/clock-triggered-tasks.rst

```diff
@@ -0,0 +1,179 @@
+.. include:: ../../../hyperlinks.rst
+   :start-line: 1
+
+Clock Triggered Tasks
+=====================
+
+This part of the Rose user guide walks you through using clock triggered tasks.
+
+This allows you to trigger tasks based on the actual time.
+
+
+Purpose
+-------
+
+.. TODO - Insert animation / diagram.
+
+The typical use of clock triggered tasks is to perform real-time data
+retrieval.
+
+In general, most tasks in a suite do not care about the wall clock time,
+simply running once their prerequisites are met. Clock triggered tasks,
+however, wait for a particular wall clock time to be reached in addition
+to their other prerequisites.
```

Comment:
This sentence isn't completely clear, at least to me, on its own - it might be better to rephrase along similar lines to the following to clarify that *even if* all of its dependencies are completed, a clock-triggered task will only run *when* the specified wall clock time is reached too, *but* that all of the dependencies must be satisfied as well, so that the set wall-clock time will pass without the task being run if there are outstanding prereq tasks (as my understanding goes - I have only read this tutorial page for this concept). Maybe my attempt at explaining this here isn't clear but I think it is definitely worth some sort of elaboration on the need for *both* wall time and dependencies to be met, especially given that there is no dependency graph in the practical example for this page so that the clock trigger acts as the only prereq there.